api/endpoint-tests/activities/approaches/put.js

```diff
@@ -15,11 +16,12 @@ tap.test(
     await db().seed.run();
 
     unauthenticatedTest('put', url(1), putGoalsTest);
+    unauthorizedTest('put', url(1), putGoalsTest);
 
     putGoalsTest.test(
       'when authenticated as a user with permission',
       async authenticated => {
-        const cookies = await login('user2@email', 'something');
+        const cookies = await login();
```

Comment:
Without args, the `login()` function uses a user with every permission and that belongs to a state.